src/components/TimeInput/__stories__/Schedule.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Story } from '@storybook/react'
+import TimeInput, { schedules } from '..'
+
+export const Schedule: Story = props => (
+  <>
+    {Object.keys(schedules).map(schedule => (
+      <div style={{ marginBottom: '16px' }}>
```

Comment:
This div should be in a decorator and you can use `Stack` instead:

```tsx
Schedule..decorators = [
  StoryComponent => (
    <Stack gap={2}>
      <StoryComponent />
    </Stack>
  ),
]
```

It will not show `div` into code snippet which will make it easier to read